src/source/bcm2835_stream.cpp

```diff
@@ -28,17 +30,86 @@ void Stream::begin(unsigned long baud, int flags)
 	}
 	fcntl(fd, F_SETFL, O_RDWR);
 
-	// Use 8 data bit, no parity and 1 stop bit
+
+	speed_t myBaud;
+	switch ( baud )

```

Comment:
The TMC2209 is maxed out at around 750kbaud and the minimum is 9600baud so I think we could leave out the unsupported values.

Reply:
Agreed!